_episodes/01-introduction.md

```diff
@@ -232,11 +241,53 @@ print(energy_kJ)
 ```
 {: .output}
 
-## A note about jupyter notebooks
-If you use the jupyter notebook for your python interpreter, the notebook only executes the current code block.  This can have several unintended consequences. If you change a value and then go back and run an earlier code block, it will use the new value, not the first defined value, which may give you incorrect analysis.  Similarly, if you open your jupyter notebook later, and try to run a code block in the middle, it may tell you that your variables are undefined, even though you can clearly see them defined in earlier code blocks.  But if you didn't re-run those code blocks, then python doesn't know they exist.  
+## Making choices: logic Statements
+Within your code, you may need to evaluate a variable and then do something if the variable has a particular value.  This type of logic is handled by an `if` statement.  In the following example, we only append the negative numbers to a new list.  
+```
+negative_numbers = []
+for number in energy_kJ:
+    if number<0:
+        negative_numbers.append(number)
 
-## Check Your Understanding
+print(negative_numbers)
+```
+{: .language-python}
+```
+[-56.0656, -11.296800000000001]
+```
+{: .output}
```

Comment:
Also introduce `else` here. What if we want one list of negative numbers and one list for positive numbers? Can also introduce `elif`.

Add logic as an objective to beginning of lesson.

Reply:
`if`, `elif` and `else` could be used for positive, negative, and zero.

Start with example of saving negative numbers, then add `else` for positive numbers. Then show if we use `if` and `else`, that 0 will be added to one of these lists - introduce `elif` (just an idea)